tests/api/V8/GetModulesMetaCest.php

```diff
@@ -36,9 +36,9 @@ public function shouldWork(ApiTester $I, Example $example)
             [
                 'data' =>
                     [
-                        'Accounts' => [
-                            'label' => 'Accounts',
-                            'access' => ['access']
+                        'type' => 'modules',
+                        'attributes' => [
```

Comment:
Just to note, this isn't actually a fix. Just wanted to see if Travis would pass :).